Integrations/integration-email-sender.yml

```diff
@@ -0,0 +1,349 @@
+        attachCIDs = argToList(demisto.getArg('attachCIDs'))
+        for i, aid in enumerate(attachIDs):
+            try:
+                fileRes = demisto.getFilePath(aid)
+                path = fileRes['path']
+                if len(attachNames) > i and attachNames[i]:
+                    filename = attachNames[i]
+                else:
+                    filename = fileRes['name']
+                if len(attachCIDs) > i and attachCIDs[i]:
+                    cid = attachCIDs[i]
+                else:
+                    cid=''
+                maintype, subtype = guess_type(filename)
+                if maintype == 'text':
+                    with open(path) as fp:
+                        data = fp.read()
+                else:
+                    with open(path, 'rb') as fp:
+                        data = fp.read()
+                attachments.append({
+                    'name': filename,
+                    'maintype': maintype,
+                    'subtype': subtype,
+                    'data': data,
+                    'cid': cid
+                })
+            except:
+                return_error('Entry %s is not valid or is not a file entry' % (aid))
+        tf = {
+            'name': demisto.getArg('transientFile'),
+            'data': demisto.getArg('transientFileContent'),
+            'cid': demisto.getArg('transientFileCID')
+        }
+        if tf['name'] and tf['data']:
+            maintype, subtype = guess_type(tf['name'])
+            attachments.append({
+                'name': tf['name'],
+                'maintype': maintype,
+                'subtype': subtype,
+                'data': tf['data'],
+                'cid': tf['cid']
+            })
+        i = 1
+        tfname = demisto.getArg('transientFile.%d' % (i))
+        while tfname:
+            tfc = demisto.getArg('transientFileContent.%d' % (i))
+            tfi = demisto.getArg('transientFileCID.%d' % (i))
+            if tfname and tfc:
+                maintype, subtype = guess_type(tfname)
+                attachments.append({
+                    'name': tfname,
+                    'maintype': maintype,
+                    'subtype': subtype,
+                    'data': tfc,
+                    'cid': tfi
+                })
+            i += 1
+            tfname = demisto.getArg('transientFile.%d' % (i))
+        return attachments
+
+
+    if demisto.command() == 'test-module':
+        msg = MIMEText('This is a test mail from Demisto\nRegards\nDBot')
+        msg['Subject'] = 'Test mail from Demisto'
+        msg['From'] = FROM
+        msg['To'] = FROM
+        SERVER.sendmail(FROM, [FROM], msg.as_string())
+        SERVER.quit()
+        demisto.results('ok')
+    elif demisto.command() == 'send-mail':
+        # Collect all parameters
+        to = argToList(demisto.getArg('to'))
+        cc = argToList(demisto.getArg('cc'))
+        bcc = argToList(demisto.getArg('bcc'))
+        subject = demisto.getArg('subject')
```

Comment:
you can use `subject = demisto.getArg('subject') or ''`

Reply:
Yup, right!